lib/ui/compositing.dart

```diff
@@ -133,10 +133,10 @@ class SceneBuilder extends NativeFieldWrapperClass2 {
   /// Rasterization will be clipped to the given shape.
   ///
   /// See [pop] for details about the operation stack.
```

Comment:
Please document the arguments in the docs here

Reply:
Done